### PR TITLE
History persistence updates

### LINKED ISSUE #2508: Employment - Additional periods answer not staying
Employment: 
- [x] Additional periods of activity question. Yes/No answer not being saved

- [x] Last question in Employment not being saved

screen shot 2018-01-25 at 2 51 17 pm

screen shot 2018-01-25 at 2 52 22 pm

Schools:

- [x] School name not being saved
screen shot 2018-01-25 at 2 53 10 pm

### PR DESCRIPTION
**Pull request by floresj**
Resolves #2508.

- Additional periods of activity question. Yes/No answer not being saved
- Last question in Employment not being saved
- School name not being saved

### CODE REVIEW

File: src/schema/section/history-education.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const historyEducation = (data = {}) => {
       Item: {
         Dates: form.daterange(xitem.Dates),
         Type: form.radio(xitem.Type),
-        Name: form.name(xitem.Name),
+        Name: form.text(xitem.Name),
```
Comment (bmallred) [inline src/schema/section/history-education.js]: :+1:

**Comment by floresj** [comment]
I see the error messages coming across too. My information appears to still persist even with the errors.

I'm thinking whats happening is in the `db.go` file, specifically at https://github.com/truetandem/e-QIP-prototype/blob/master/api/db/db.go#L98,  the Save method attempts to do an insert. If there's an error, we then perform an update. Those error messages look like they're coming directly from postgres so that's probably why they're showing up on my end but still persisting because it then does an update which succeeds.

**Comment by bmallred** [comment]
That makes since, but mine **ends** with an error and no subsequent update is performed.

**Comment by floresj** [comment]
The fact that it's not working on your end is puzzling me!

**Pull request merged**